lighthouse-core/report/html/report-styles.css

```diff
@@ -15,41 +15,56 @@
  * limitations under the License.
  */
 
+/*
+  Naming convention:
+
+  If a variable is used for a specific component: --{component}-{property name}-{modifier}
+
+  Both {component} and {property name} should be kebab-case. If the target is the entire page,
+  use 'report' for the component. The property name should be not be abbreviated. Use the
+  property name the variable is intended for - if it's used for multiple, a common descriptor
+  is fine (ex: 'size' for a variable applied to 'width' and 'height'). If a variable is shared
+  across multiple components, either create more variables or just drop the "{component}-"
+  part of the name. Append any modifiers at the end (ex: 'big', 'dark').
+
+  For colors: --color-{type}-{hue}
```

Comment:
seems like you mean intensity/lightness/saturation here instead of `hue`, `type` is currently the hue :)

Reply:
Yes, thanks!

Reply:
I couldn't find an MD lingo for describing those tags (`700`, `A700`, ...)